code/modules/mechs/equipment/utility.dm

```diff
@@ -207,6 +207,15 @@
 	// OCCULUS EDIT -- durability was not being properly applied because material during this proc is null
 	//durability = 2 * (material ? material.integrity : 1)
 
+///// OCCULUS EDIT BEGIN
+// Drills crafted from the crafting menu actually call Created().
+// var/creator is a dummy var to avoid any issues with the call.
+
+/obj/item/weapon/material/drill_head/Created(var/creator)
```

Comment:
You don't actually need var/creator here! It isn't used by anything else in the code so it'd have no effect here.